package.json

```diff
@@ -17,37 +19,37 @@
     "azure",
     "ansible module"
   ],
+  "repository": {
+    "url": "https://github.com/VSChina/vscode-ansible"
+  },
+  "homepage": "https://github.com/VSChina/vscode-ansible/blob/master/README.md",
   "activationEvents": [
-    "onCommand:vsc-extension-ansible.ansible-playbook",
-    "onCommand:vsc-extension-ansible.ansible-commands",
-    "onCommand:vsc-extension-ansible.ansible-terminal",
-    "onCommand:vsc-extension-ansible.ansible-playbook-in-terminal"
+    "onCommand:vscode-ansible.ansible-playbook",
+    "onCommand:vscode-ansible.ansible-commands",
+    "onCommand:vscode-ansible.ansible-playbook-in-terminal",
```

Comment:
i wonder if we should keep "just terminal" option as well.
that should display either menu to choose terminal or cloudshell, or just default (should be in config)